pkg/kwok/controllers/pod_controller_test.go

```diff
@@ -49,6 +50,22 @@ func TestPodController(t *testing.T) {
 				NodeName: "node0",
 			},
 		},
+		&corev1.Pod{
```

Comment:
It is better to test the complete allocation and recycling behavior of Node's PodCIDR and default CIDR in a new test function